File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-04/plagiarized/L3/05/pengulangan.ts
import { Console } from 'console';

class inputoutput {
  public static main(args: string[]): void {
    let m: number = 1;
    // print seusatu
    console.log('Miles\t\tKilometers');
    console.log('-------------------------------');
    // pake pengulangan while
    while (m <= 10) {
      console.log(`${m}\t\t${m * 1.609}`);
      m++;
      // ditambahin iteratornya
    }
  }
}

File: datasets/IR-Plag/intellij-original-java-rest-kotlin/case-04/plagiarized/L1/05/pengulangan.kt
package IR

internal object inputoutput {
    @JvmStatic
    fun main(args: Array<String>) {
        // print seusatu
        println("Miles\t\tKilometers")
        println("-------------------------------")
        // pake pengulangan while
        var miles = 1
        while (miles <= 10) {
            println(miles.toString() + "\t\t" + miles * 1.609)
            miles++
            //ditambahin iteratornya
        }
    }
}